app/javascript/components/UserProfileMenu/index.jsx

```diff
@@ -95,6 +98,8 @@ const UserProfileMenu = ({ offices, location, router, togglePopover }) => {
         i18n.changeLanguage(selectedItem.language.value, () => {
           // TODO: Handle callback (error/success)
         })
+        const selectedLanguage = R.find(R.propEq("languageCode", selectedItem.language.value), languages)
```

Comment:
This is repeated in line 54, it might be worth making a function:
e.g.
```
const getLanguageByCode = (code) => R.find(R.propEq("languageCode", code), languages)
```